src/AccessibilityInsights.SharedUx/Controls/WhatsNew/WhatsNewControl.xaml

```diff
@@ -31,7 +31,7 @@
                 <RowDefinition Height="Auto"/>
                 <RowDefinition Height="Auto"/>
             </Grid.RowDefinitions>
-            <fabric:FabricIconControl VerticalAlignment="Center" GlyphName="SearchAndApps" GlyphSize="Custom" FontSize="18" Foreground="{DynamicResource ResourceKey=IconBrush}" Grid.Column="1" Grid.Row="0"/>
+            <fabric:FabricIconControl VerticalAlignment="Center" GlyphName="SearchAndApps" GlyphSize="Custom" FontSize="18" Foreground="{DynamicResource ResourceKey=IconBrush}" Grid.Column="1" Grid.Row="0" ShowInControlView="False"/>
```

Comment:
![screenshot of what's new page, several decorative icons alongside their feature descriptions](https://user-images.githubusercontent.com/7775527/64988947-65b12600-d881-11e9-8f50-0f881d103269.png)